com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/errors.py

```diff
@@ -0,0 +1,2 @@
+class ViewNotFoundError(Exception):
+    """Raised when the SWS view element cannot be located."""
```

Comment:
I'm not convinced, I don't believe 'SWS' is used in the docs for the REST api.

https://www.ibm.com/support/knowledgecenter/en/SSCRJU_4.0.0/com.ibm.streams.restapi.doc/doc/restapis.html